src/cap-description.adoc

```diff
@@ -468,6 +469,8 @@ if ( (E < (CAP_MAX_E - 1)) & (t[MXLEN: MXLEN - 1] - b[MXLEN - 1] > 1) )
 That is, invert the most significant bit of _t_ if the decoded length of the
 capability is larger than E.
 
+NOTE: Capabilities with E=CAP_MAX_E have infinite bounds.
+
```

Comment:
@PRugg-Cap @PeterRugg @tariqkurd-repo : I modified the definition of "infinite bounds". Could you please confirm that this note and the previous paragraph are as expected? Thanks!

Reply:
Looks fixed, thanks! For some reason this preview is showing me something stale, but the most recent thing I saw that includes the malformed note looks right.